readthedocs/search/tests/data/generic/out/basic.json

```diff
@@ -6,7 +6,22 @@
       {
         "id": "",
         "title": "Title of the page",
-        "content": "Content of the body."
+        "content": "Content of the body. Love To code is to love No section for me This term does not have an ID so it's not gonna get its own section Code Docs Code and docs are like love, they are 4-letter words."
```

Comment:
This doesn't look correct, we are indexing the content from the dls here again.

Reply:
Oh interesting, actually this block should remove the stuff that's indexed, right?

But a quick hack would be to move the `<dl>` stuff above since it already removes indexed nodes :+1:

Reply:
nice find!

Reply:
Shuffled the order of the parsing, this looks good now :+1: